src/UserInterface.php

```diff
@@ -21,5 +21,5 @@ public function getIdentity() : string;
      *
      * @return string[]
      */
-    public function getUserRoles() : array;
+    public function getUserRoles() : iterable;
```

Comment:
It should be nice to add some test to cover that change.

Reply:
What do you propose? No current test for this interface exists that I can tell? Happy to push more commits though.